Tools/scripts/docker_build.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [ $# -ne 1 ]
+  then
```

Comment:
```suggestion
if [ $# -ne 1 ]; then
```

for no particular reason apart from that's what we do most places.